swagger/index.js

```diff
@@ -95,6 +95,7 @@ module.exports = yeoman.Base.extend({
         basePath = basePath.substring(1);
       }
 
+      /*
```

Comment:
Why is this block commented out? Either fix the problem or remove the unused code entirely.